### ISSUE TITLE
PySide2 is a confusing name

### ISSUE THREAD

**Issue by SteveDoyle2** [PySide/pyside2#26]
Since PySide2 implements Qt5, I think it'd be more clear if PySide2 was called PySide5.

**Comment by leeoo** [comment]
+1

**Comment by en90** [comment]
+1

**Comment by IronManMark20** [comment]
This is a rather low priority. We want to first work on getting some product that works out there. Then talking about name changes is okay. I think a good time to bring this back up would be when we hit beta perhaps (but may be another time)

**Comment by leycec** [comment]
Actually, this should be a top priority. A few painfully obvious reasons why:

* **It's low-hanging fruit.** At least, it is _now_. Once upstream Linux distributions and OS X package managers begin officially packaging this project and other packages begin transitively requiring this project, once upstream Python projects begin requiring this project as a hard `setup.py` requirement, and once PyPI begins officially listing this project, a name change will no longer be low-hanging fruit; it will essentially be infeasible.
* **Queryability.** Equivalently, search engine optimization (SEO). The later you wait on a name change, the greater the search engine pain you foist upon your userbase. Search engine hits will be split between the "pyside2" and "pyside5" search terms, significantly complicating searching _and_ effectively obliterating any SEO (e.g., page rank) thus accrued from the increasingly used "pyside2" search term.
* **Rationality.** PySide2 makes no rational sense as a project name and consequently gives a poor impression to would-be early adopters and contributors. (_That's bad._)

If this is going to happen (...and it should), it absolutely needs to happen as a soon as possible. A formal name change is _not_ deferrable to a future milestone. It's time-sensitive. It can't wait.

Your window of opportunity is closing fast.

**Comment by Juanlu001** [comment]
I find @leycec points very spot on and also agree on calling this project PySide5.

**Comment by ctismer** [comment]
@IronManMark20 @SteveDoyle2  seconded. I think we should rename you as well, as we are at it ;-)

**Comment by dekekincaid** [comment]
I don't understand why this matters.  All projects have their own versioning scheme which are separate from QT itself.  Even QT projects are not on the same version as QT itself: QT Creator 3.5 , QT Quick 2, Qmake, QML, etc... same with 3rd party projects: WX Widgets, PythonQT (Pyqt is version 5 but that is a coincidence and causes other issues as outlined below).  Pyside versions may not always be in lockstep with the version of QT they support, IE if QT6 comes out, then Pyside5 may still very well work with it, do you version it to Pyside6 for the hell of it even though there aren't any new features but still have it compatible with QT5 just to keep the numbering consistent even though there may not be enough code difference to move to a new version?  

The Pyqt project actually runs into these issues and confusions by happen to name it PyQt5 where even though that is it's name it doesn't imply qt5 only support.  Read the "An Explanation of Version Numbers" section of the bottom of this page:
http://pyqt.sourceforge.net/Docs/PyQt5/introduction.html#pyqt5-components

I vote for leaving it at pyside 2.

**Comment by KronosKoderS** [comment]
I've got a question, will PySide2 support QT4 bindings?

**Comment by IronManMark20** [comment]
We are working on Qt5 only. We direct people who have bugs, etc to the original PySide for Qt4 related problems.

**Comment by KronosKoderS** [comment]
As a reference: 

> Starting with PyQt5 the version number of PyQt5 is tied, to a certain extent, to the version of Qt v5. This is based on the following assumptions:
>
> * All parts of the Qt API will be supported throughout the life of Qt v5 even though some may be marked as deprecated or obsolete at some point.
> * When new parts of the Qt API are introduced the minor number of the version will be increased and the maintenance number will be reset to 0.
> 
> Therefore, for PyQt5 v5.n.* the following are true:
> 
> * It will build against any version of Qt v5, but will not support any new features introduced in Qt v5.n+1 or later.
> * It will support all the features of supported modules of Qt v5.n or earlier.
> * Support for new modules may be added to PyQt5 at any time. This would result in a change of maintenance number only.
> * The maintenance numbers of PyQt5 and Qt v5 are entirely unrelated to each other.

> So, for example, PyQt5 v5.1 will build against Qt v5.2 but will not support any new features introduced in Qt v5.2. PyQt5 v5.1 will support all the features of supported modules of Qt v5.0 and those new features introduced in Qt v5.1.
